pyanaconda/addons.py

```diff
@@ -88,6 +91,7 @@ def execute(self, storage, ksdata, users, payload):
         for v in self.__dict__.values():
             if hasattr(v, "execute"):
                 progress_message(N_("Executing %s addon") % v.name)
+                log.debug("Executing %s addon", v.name)
```

Comment:
I think that the progress message above is logged as well, so it is not necessary to log it again.

Reply:
Fixing now.